static/sass/layout/_nav.scss

```diff
@@ -11,15 +11,20 @@ $nav-locations:
   'contact',
 ;
 
+.nav-link {
+  #{$link} {
+    display: inline-block;
+    padding: size('half-shim');
+    text-decoration: none;
+  }
+}
+
 @each $location in $nav-locations {
   [data-location='#{$location}'] {
     [data-active='#{$location}'] {
-      &:link,
-      &:visited {
-        @include wrap-content('[' ']');
-        color: inherit;
-        font-weight: bold;
-        text-decoration: none;
+      #{$link} {
+        color: color('action');
+        font-style: italic;
```

Comment:
I know this is part of the design, but it is always weird to me when a nav bar shifts from page to page since the width of the word is now different when italicized.